test/runner.zig

```diff
@@ -208,33 +217,25 @@ fn mainTerminal() void {
 		testing.log_level = .warn;
 
 		const test_node = root_node.start(test_fn.name, 0);
-		if(!have_tty) {
-			std.debug.print("{d}/{d} {s}...", .{i + 1, test_fn_list.len, test_fn.name});
-		}
+
+		const padding_length: usize = max_length - test_fn.name.len + 3;
+		const padding = padding_buffer[0..padding_length];
+
+		std.debug.print("{d: >4}/{d: <4} {s}{s}", .{i + 1, test_fn_list.len, test_fn.name, padding});
```

Comment:
I would suggest to use a fixed padding length, to prevent long test names from making every line wrap around if the terminal is too small. My default terminal size is 80, so I'd suggest 65 characters.
This would allow you to use the formatters padding option here as well:
```suggestion
		std.debug.print("{d: >4}/{d: <4} {s:.<65}", .{i + 1, test_fn_list.len, test_fn.name});
```